src/RRule.php

```diff
@@ -2062,17 +2062,19 @@ static protected function i18nLoad($locale, $fallback = null, $use_intl = null,
 	 *
 	 * Available options
 	 *
-	 * | Name              | Type    | Description
-	 * |-------------------|---------|------------
-	 * | `use_intl`        | bool    | Use the intl extension or not (autodetect)
-	 * | `locale`          | string  | The locale to use (autodetect)
-	 * | `fallback`        | string  | Fallback locale if main locale is not found (default en)
-	 * | `date_formatter`  | callable| Function used to format the date (takes date, returns formatted)
-	 * | `explicit_inifite`| bool    | Mention "forever" if the rule is infinite (true)
-	 * | `dtstart`         | bool    | Mention the start date (true)
-	 * | `include_start`   | bool    |
-	 * | `include_until`   | bool    |
-	 * | `custom_path`     | string  |
+	 * | Name                  | Type    | Description
+	 * |-----------------------|---------|------------
+	 * | `use_intl`            | bool    | Use the intl extension or not (autodetect)
+	 * | `locale`              | string  | The locale to use (autodetect)
+	 * | `fallback`            | string  | Fallback locale if main locale is not found (default en)
+	 * | `date_formatter`      | callable| Function used to format the date (takes date, returns formatted)
+	 * | `timeofday_formatter` | callable| Function used to format the timeofday (takes date, returns formatted)
+	 * | `explicit_inifite`    | bool    | Mention "forever" if the rule is infinite (true)
+	 * | `dtstart`             | bool    | Mention the start date (true)
+	 * | `include_start`       | bool    |
+	 * | `include_timeofday`   | bool    |
+	 * | `include_until`       | bool    |
+	 * | `custom_path`         | string  |
```

Comment:
Why are all the lines in the diff? Only the last one changed - is this an indentation change or something?

Reply:
Yes this is only because the new option names are longer so the table ended up wider.